src/main/java/bg/startit/hackathon/airquiality/model/User.java

```diff
@@ -49,6 +49,21 @@
   @Column(unique = true)
   private String email;
 
+  public User(@Email @NotNull String email)
+  {
+    this.email = email;
+  }
+
+  public String getEmail()
+  {
+    return email;
+  }
+
+  public void setEmail(String email)
+  {
+    this.email = email;
+  }
+
```

Comment:
This is already here! Just install lombok plugin for IDEA!
Please remove all these lines.